<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringFrame</class>
 <widget class="QWidget" name="StringFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>22</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="delToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../property.qrc">
       <normaloff>:/objectcontroller/images/Remove.png</normaloff>:/objectcontroller/images/Remove.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="upToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../property.qrc">
       <normaloff>:/objectcontroller/images/Up.png</normaloff>:/objectcontroller/images/Up.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="downToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../property.qrc">
       <normaloff>:/objectcontroller/images/Down.png</normaloff>:/objectcontroller/images/Down.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="valueEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../property.qrc"/>
 </resources>
 <connections/>
</ui>
